src/main/webapp/index.html

```diff
@@ -50,17 +50,18 @@
       </div>
       <div class="inline">
         <form id="input-form">
-          <input type="text" class="input-text" id="input-text" name="input-text" placeholder="Enter your postcard text.."> 
+          <input type="text" class="input-text" id="input-text" maxlength="300" name="input-text" onkeydown="return (event.keyCode!=13);" placeholder="Enter your postcard text.."> 
```

Comment:
isn't the max length of characters for the postcard 310?
this._maxMessageLength = 310;

also below

Reply:
hmm Andrei told me it's 300. I'll check on that